sql/core/src/main/scala/org/apache/spark/sql/execution/PartitionedFileUtil.scala

```diff
@@ -69,8 +69,10 @@ object PartitionedFileUtil {
         b.getHosts -> (b.getOffset + b.getLength - offset).min(length)
```

Comment:
I might update the comment above this, while you're here, to say that it handles the case where the fragment is fully contained in the block. It would help future readers; it took me some time to work it out.

Reply:
This is still worth updating.

Reply:
Updated.